app/src/main/res/layout/layout_item_history.xml

```diff
@@ -6,7 +6,7 @@
 
     <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="76dp"
```

Comment:
I dont think makign it a constant would work for all devices.

Ideally wrap content should work. You can try increasing the padding for inner layouts, but making the height a constant is not a correct solution to fix this issue